src/template/js/sdk/Events/index.js

```diff
@@ -129,6 +140,34 @@ const listen = function(...args) {
   return doListen(module, event, callback, false)
 }
 
+const provide = function(...args) {
```

Comment:
Hmm... I was assuming the generic `provide` method would live in a new Capabilities module. We're only using events to implement this cause of our desire to use RPC and only have one client and one server. Seems irrelevant to the App.

Can we move this to a new Capabilities.provide method, w/out causing a headache? It would be a static module, I think. No need for a modules/capabilities.json, it'd just be in src/template/js/sdk/Capabilities/index.js